sample-app/package.json

```diff
@@ -134,7 +134,7 @@
       "<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}"
     ],
     "testEnvironment": "jsdom",
-    "testRunner": "/Users/oshi/answers-headless/react-test-site/node_modules/jest-circus/runner.js",
+    "testRunner": "./node_modules/jest-circus/runner.js",
```

Comment:
I know we're not using this, but should it be `<rootDir>` instead?

Reply:
either works (also tested cd-ing into a nested folder)